packages/desktop/views/dashboard/send-flow/views/TransactionSummaryView.svelte

```diff
@@ -104,7 +105,12 @@
             busy = true
             modifyPopupState({ preventClose: true })
             if (isSourceNetworkLayer2) {
-                await signAndSendTransactionFromEvm(preparedTransaction, chain, $selectedAccount, true)
+                const signedTransaction = await signEvmTransaction(preparedTransaction, chain, $selectedAccount)
```

Comment:
The error throwing can be moved into the function `signEvmTransaction` here and in `EvmTransactionFromDappPopup`.